# reasonable parameters to find planes of @Work workspaces
scene_segmentation:
    voxel_leaf_size: 0.009
    voxel_filter_field_name: z
    voxel_filter_limit_min: -0.15
    voxel_filter_limit_max: 0.3
    enable_passthrough_filter: false
    passthrough_filter_field_name: x
    passthrough_filter_limit_min: 0.0
    passthrough_filter_limit_max: 0.8
    normal_radius_search: 0.03
    sac_max_iterations: 1000
    sac_distance_threshold: 0.01
    sac_optimize_coefficients: true
    sac_eps_angle: 0.09
    sac_normal_distance_weight: 0.05
    prism_min_height: 0.01
    prism_max_height: 0.10
    outlier_radius_search: 0.03
    outlier_min_neighbors: 20
    cluster_tolerance: 0.02
    cluster_min_size: 25
    cluster_max_size: 20000
    cluster_min_height: 0.011
    cluster_max_height: 0.09
    cluster_max_length: 0.25
    cluster_min_distance_to_polygon: 0.04
    octree_resolution: 0.0025
    object_height_above_workspace: 0.052
